testgrid/specs/os-full.yaml

```diff
@@ -54,17 +54,13 @@
     curl -L -o centos2ol.sh https://raw.githubusercontent.com/oracle/centos2ol/main/centos2ol.sh
     chmod +x centos2ol.sh
     bash centos2ol.sh -r
-- id: rhel-9
-  name: RHEL
-  version: "9.x"
-  vmimageuri: https://download.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud-Base.latest.x86_64.qcow2
-- id: centos-9
-  name: CentOS
-  version: "9.x"
-  vmimageuri: https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-20230227.0.x86_64.qcow2
-- id: ol-9
+- id: rocky-91
+  name: Rocky Linux
+  version: "9.1"
+  vmimageuri: https://download.rockylinux.org/pub/rocky/9/images/x86_64/rocky-91-GenericCloud-Base-9.1-20230215.0.x86_64.qcow2
+- id: ol-91
   name: Oracle Linux
-  version: "9.x"
+  version: "9.1"
```

Comment:
I'm so sorry
this one actually is 9.x (the oracle init script goes to the latest version)
see
```
- id: ol-8x
  name: Oracle Linux
  version: "8.x"
  vmimageuri: https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.4.2105-20210603.0.x86_64.qcow2
```